spec/spec_helper.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
-
+require 'simplecov'
+SimpleCov.start do
+  SimpleCov.add_filter '/spec'
```

Comment:
Trying to think of other things that fall outside the realm of test coverage. Are things in the `bin/` and `db/` showing up in the report (I haven’t run it locally yet - on my phone)? I don’t think we would be writing tests to cover those.

Reply:
@cattywampus here's what I'm seeing when I run it locally:
![image](https://user-images.githubusercontent.com/47750744/67044003-50861c00-f0f9-11e9-91b5-08bcac4f7cb4.png)